tables/top_level_geographies.md

```diff
@@ -1 +1,45 @@
+# tables/top_level_geographies
 
+The `top_level_geographies` table returns information regarding the individual `top-level geographies` which are identified via an `id`.
+
+The following `JOIN` queries can be carried out:
+
+- `geography_type_id` on the [geography_groupings](geography_groupings.md) table using `geography_type_id`.
+- `top_level_geography_id` on the [geography_areas](geography_areas.md) table using `id`.
+
+## What are top-level geographies?
+
+A `top-level geography` refers to the first (highest) geography level selectable. This is usually chosen at the start of a search by an end-user.
+
+## Example use
+
+If searching for a combination of everyone in `Scotland` shown by `age`, you would first select a top-level geography of `Scotland` before selecting a lower-level geography (such as `Local Authority`), and finally the `age` topic.
+
+## Schema
+
```

Comment:
it's probably worth mentioning in the schema that the `id` is also a foreign key, that relates to the tables `geography_areas` and `geography_groupings`